protocol/xmrmaker/instance_test.go

```diff
@@ -101,12 +108,12 @@ func newBackendAndNet(t *testing.T) (backend.Backend, *mockNet) {
 	rdb.EXPECT().PutCounterpartySwapKeys(gomock.Any(), gomock.Any(), gomock.Any()).Return(nil).AnyTimes()
 	rdb.EXPECT().DeleteSwap(gomock.Any()).Return(nil).AnyTimes()
 
-	extendedEC, err := extethclient.NewEthClient(context.Background(), env, common.DefaultEthEndpoint, pk)
+	extendedEC, err := extethclient.NewEthClient(ctx, env, common.DefaultEthEndpoint, pk)
```

Comment:
Not related to this PR, but should we have changed `common.DefaultEthEndpoint` to use the `http` endpoint after the recent pull request removing the websockets dependency?

Reply:
yeah we can, either one should still work though